jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -23,19 +24,18 @@ class LogStreamsStreamColumn : ColumnInfo<LogStream, String>(message("cloudwatch
 }
 
 class LogStreamsDateColumn : ColumnInfo<LogStream, String>(message("cloudwatch.logs.last_event_time")) {
-    override fun valueOf(item: LogStream?): String? {
-        val timestamp = item?.lastEventTimestamp() ?: return null
-        return DateFormatUtil.getDateTimeFormat().format(timestamp)
-    }
+    private val renderer = ResizingDateColumnRenderer()
+    override fun valueOf(item: LogStream?): String? = item?.lastEventTimestamp()?.toString()
 
     override fun isCellEditable(item: LogStream?): Boolean = false
+    override fun getRenderer(item: LogStream?): TableCellRenderer? = renderer
 }
 
 class LogGroupTableSorter(model: ListTableModel<LogStream>) : TableRowSorter<ListTableModel<LogStream>>(model) {
```

Comment:
What is behavior around scrolling if we change the sort and load more data?

Reply:
This table can't currently load more data: it loads in all of the log groups. In a later PR the table needs to be extended to do that, but for right now I have no idea what would happen.

Reply:
That makes sense...I'm getting my PRs confused